### ISSUE TITLE
Feature Suggestion: Washington Post bypass w/ Adblock

### ISSUE THREAD

**Issue by zacaytion** [iamadamdev/bypass-paywalls-firefox#312]
Thanks for your awesome extension!

Since WaPo has started detecting adblock, I've had trouble accessing the site.
However I recently discovered that if you change the subdomain from `www` to `beta` and add `outputType=amp` as url Params, the page loads with no problem. 

It would be great if the extension could redirect users to the beta subdomain. Happy to look into implementing if it's something that seems feasible to add to the extension

**Comment by VernonStow** [comment]
Related: Washington Post randomly showing anti-ad block warning with uBlockOrigin and bypass-paywalls-firefox enabled https://github.com/uBlockOrigin/uAssets/issues/6339

**Comment by iamadamdev** [comment]
Yes that sounds like a nice improvement. Personally I am not getting the paywall so it would be hard for me test.
Can you make the changes and submit a PR?

thanks
